arangodb-net-standard.Test/TransactionApi/TransactionApiClientTest.cs

```diff
@@ -14,8 +14,8 @@ public class TransactionApiClientTest: IClassFixture<TransactionApiClientTestFix
         public TransactionApiClientTest(TransactionApiClientTestFixture fixture)
         {
             _adb = fixture.ArangoDBClient;
-            _adb.Collection.PutCollectionTruncateAsync(fixture.TestCollection1).Wait();
-            _adb.Collection.PutCollectionTruncateAsync(fixture.TestCollection2).Wait();
+            _adb.Collection.TruncateCollectionAsync(fixture.TestCollection1).Wait();
```

Comment:
I've just noticed these `.Wait()` thanks to the changes. Should we also implement `IAsyncLifetime` for that test class?

Otherwise maybe change the `.Wait()` to `GetAwaiter().GetResult()`?

Reply:
I don't have an opinion on that, if it works though, why fix it?